### ISSUE TITLE
[ExtraUtils2 1.4.1] 1.11.2 Server crash when someone goes to Deep Dark

### ISSUE THREAD

**Issue by shortybsd** [rwtema/extrautilities_old_issues#1470]
MC 1.11.2
Forge 2284
ExtraUtils2 1.4.1

Anytime anyone goes through the Deep Dark portal it crashes the server with the following error:
https://pastebin.com/SY5yt5sZ

I deleted the world for Deep Dark, loaded back up and same issue.

**Comment by Endmateria** [comment]
@shortybsd  Yeah, the Deep Dark is experiencing technical difficulties at the moment.  I believe the tickrate of the entire dimension is messed up for some reason or other.  I opened an issue on the matter a couple of months back...  https://github.com/rwtema/extrautilities/issues/1311
